# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lsof/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lsof_4.78_src/lib/Makefile.skel..vanilla	2007-06-16 21:07:49.248374495 +0200
+++ lsof_4.78_src/lib/Makefile.skel	2007-06-16 21:09:06.092753607 +0200
@@ -41,8 +41,8 @@
 all:	${LIB}
 
 ${LIB}:	${OBJ}
-	${AR}
-	${RANLIB}
+	${AR} cr ${LIB} ${OBJ}
+	${RANLIB} ${LIB}
 
 clean:	FRC
 	rm -f ${LIB} ${OBJ} errs Makefile.bak a.out core
